src/postamble.js

```diff
@@ -163,6 +163,7 @@ dependenciesFulfilled = function runCaller() {
   if (!Module['calledRun']) dependenciesFulfilled = runCaller; // try this again later, after new deps are fulfilled
 }
 
+#if HAS_MAIN
 Module['callMain'] = Module.callMain = function callMain(args) {
```

Comment:
`Module['callMain']` and `Module.callMain` are same in JS, can we remove the latter part?

Reply:
Yes, I'm actually not sure why we have this pattern. I suspect it is very old, and either worked around a closure compiler bug or a bug in our understanding of closure compiler ;) We should remove it across the codebase, leaving only `Module['X']`.